lib/cleanup.js

```diff
@@ -1,12 +0,0 @@
-function removeLeadingZerosFromStreet(token) {
```

Comment:
I'd leave it as is for now. It's easy enough to deliberately split it out when we start tackling the implementation of this OA specific cleanup/validation module.